apps/web/src/pages/templates/filter/Filters.tsx

```diff
@@ -26,22 +27,27 @@ export const Filters = ({ step }: { step: StepEntity | null }) => {
   );
 };
 
-interface IFilter {
-  on?: 'payload' | 'subscriber';
-  field?: string;
-  value?: string;
-  operator?: BuilderFieldOperator;
-}
-
-export const Filter = ({ filter }: { filter: IFilter }) => {
+export const Filter = ({ filter }: { filter: FilterParts }) => {
   const { colorScheme } = useMantineColorScheme();
 
+  const filterValue = useMemo(() => {
+    let value = filter.value;
+    if (filter.on === 'isOnline') {
+      if (filter.value === true) {
+        value = 'Yes';
+      }
+      if (filter.value === false) {
+        value = 'No';
+      }
+    }
+
```

Comment:
Manually checking for `filter.value === true`, `filter.value === false` coz,
when the user has not selected any value, `filter.value` will be undefined or empty